### PR TITLE
async scout constructor

### PR DESCRIPTION
**Pull request by Sarmisuper**

### CODE REVIEW

**Review by mwj427 — APPROVED** [review]

**Review by Juhe20 — APPROVED** [review]
I agree!

File: HotelPriceScout/Data/Model/BookingSite.cs [hunk 1]
```diff
@@ -53,12 +53,12 @@ public void CreatePriceScraper()
...
     }
 
-    private IEnumerable<Hotel> CreateHotels(List<string> hotelsStrings)
+    private IEnumerable<Hotel> CreateHotels(Dictionary<string, string> hotelsStrings)
     {
       List<Hotel> hotels = new List<Hotel>();
-      foreach(string hotel in hotelsStrings)
+      foreach(KeyValuePair<string, string> hotel in hotelsStrings)
         {
-          //hotels.Add(new Hotel(hotel));
```
Comment (WassawRoki) [inline HotelPriceScout/Data/Model/BookingSite.cs:61]: remove zombie code

**Review by WassawRoki — APPROVED** [review]
der er grønt lys her fra :D

File: HotelPriceScout/Data/Model/BookingSite.cs (hunk 1 above)
Comment (Sarmisuper) [reply HotelPriceScout/Data/Model/BookingSite.cs:61]: it is dead already

**Pull request closed without merging**